.gitlab-ci.yml

```diff
@@ -158,8 +147,6 @@ sanity_checks:
     - 'bundle exec rake pkg:compare_latest_tag'
     - 'bundle exec rake pkg:create_tag_changelog'
     - 'bundle exec puppet module build'
```

Comment:
I do not believe the extra linting should have been removed

Reply:
@lnemsick-simp as the `.gitlab-ci.yml` is standardized between modules, it's not going to be a good place to put custom checks.   I've moved these into a `rakelib/` file and configured them to run during the other `rake syntax` checks.